doto-frontend/src/components/pages/Calendar/CalendarComponent.js

```diff
@@ -12,11 +12,12 @@ import {
     DateNavigator,
     TodayButton,
     Appointments,
+    AppointmentTooltip,
 } from "@devexpress/dx-react-scheduler-material-ui";
 
-const CalendarComponent = ({ tasks }) => {
+const CalendarComponent = props => {
```

Comment:
Maybe it would be useful to have a comment above explaining what this component is used for

Reply:
Similarly for all other custom React components we made

Reply:
Lakshya is gonna do comments because if I do, there will be no comments

Reply:
I'll add it in during commenting (I'll do in the next PR).